docs/uperf.md

```diff
@@ -28,6 +33,22 @@ spec:
     runtime: 60
 ```
 
+`hostnetwork` will test the performance of the node the pod will run on.
+
+*Note:* If you want to run with hostnetwork, you will need to execute the following:
+
+```bash
+
+$ oc adm policy add-scc-to-user privileged -z benchmark-operator
```

Comment:
What's the way to do this through `kubectl` ? It'd be helpful to add that here and then put forward saying if you've oc installed then you can accomplish this through the above mentioned.

Reply:
Not sure. You can try this commit on a pure kube deployment to see if this is even needed.

Reply:
Added a note for OpenShift env. Unsure about Kube.